src/pudl/metadata/sources.py

```diff
@@ -665,6 +665,29 @@
         "license_raw": LICENSES["us-govt"],
         "license_pudl": LICENSES["cc-by-4.0"],
     },
+    "ferccid": {
+        "title": "FERC CID - Company Identifier Listing",
+        "path": "https://data.ferc.gov/company-registration/ferc-company-identifier-listing/",
```

Comment:
Oh wow, a snazzy interactive data table from FERC. Are you able to actually download from that interface easily in the archiver? or does it require playwright?

Reply:
Ah now I'm seeing your comment on the archiver PR about using the interactive dashboard link instead. I'll switch over there.

Reply:
Yeah, I think this is the right URL to use, both for the metadata and as the source for the data.